### DISCUSSION TITLE
Identifying new version of asdf in Dockerfile

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#20719] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate to open a PR when new asdf version is identified when asdf is installed in a Dockerfile

### What you tried so far.

Applying the following configuration. Not sure what to populate in `depNameTemplate`
 ```
"regexManagers": [
        {
          "fileMatch": ["^Dockerfile$"],
          "matchStrings": ["ENV ASDF_VERSION=(?<currentValue>.*?)\\n"],
          "depNameTemplate": "asdf-vm/asdf",
          "datasourceTemplate": "github-releases"
        }
    ]

```
Dockerfile:

```
ENV ASDF_VERSION=0.11.1

RUN dnf install -y \
    git openssh-clients python3-pip make unzip && \
    dnf clean all -y && \
    git clone https://github.com/asdf-vm/asdf.git /root/.asdf --branch v"${ASDF_VERSION}" --depth 1 && \
    . $HOME/.asdf/asdf.sh && \
    asdf plugin-add awscli && \
    asdf plugin-add terraform https://github.com/asdf-community/asdf-hashicorp.git && \
    asdf plugin-add terragrunt && \
    asdf install && \
    pip3 install -r /root/requirements.txt

```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
That looks ok to me. What log/problem are you seeing?

**Reply by lmilbaum** [reply]
> That looks ok to me. What log/problem are you seeing?

Didn't apply it yet. Wasn't sure if that syntax is correct.
Thank you for your feedback.
